File: app/src/main/java/com/neome/api/meta/base/dto/StudioPluginDev.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdForm;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

import java.util.Date;

@SuppressWarnings("unused")
public class StudioPluginDev extends StudioBase
{
  public Date lastUpdateTime;

  @Nullable
  public MetaIdVar packageNameVarId;

  @Nullable
  public MetaIdForm pluginConfigFormId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioPluginDev.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdForm
import com.neome.api.meta.base.Types.MetaIdVar

open class StudioPluginDev : StudioBase() {
    lateinit var lastUpdateTime: String
    var packageNameVarId: MetaIdVar? = null
    var pluginConfigFormId: MetaIdForm? = null
}
